ambry-cloud/src/main/java/com/github/ambry/cloud/azure/AzureCloudConfig.java

```diff
@@ -163,6 +288,13 @@
   @Default("false")
   public final boolean cosmosDirectHttps;
 
+  /**
+   * Azure storage account info.
+   */
+  @Config(AZURE_STORAGE_ACCOUNT_INFO)
+  @Default("")
+  public final String getAzureStorageAccountInfo;
```

Comment:
Since the StorageAccountInfo class doesn't bring any module dependencies with it, we should make the `azureStorageAccountInfo` config as `StorageAccountInfo` type instead of `String` type. That way we will know right away if the config passed is valid.

Reply:
Done